symplyphysics/laws/gravity/radial_motion/total_energy_is_radial_kinetic_energy_plus_potential_energy.py

```diff
@@ -0,0 +1,88 @@
+"""
+Radial kinetic energy plus potential energy is constant
+=======================================================
+
+The total energy of the planet is composed of its radial kinetic energy and potential
+energy. Note that the sign of the total energy determines the type of the planet's orbit:
+
+#. If :math:`E < 0`, the orbit is elliptical.
+#. If :math:`E = 0`, the orbit is parabolical.
+#. If :math:`E > 0`, the orbit is hyperbolical.
+"""
+
+from sympy import Eq
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    validate_input,
+    validate_output,
+    symbols,
+    clone_symbol,
+)
+
+planetary_mass = clone_symbol(symbols.basic.mass, "planetary_mass")
+"""
+The :attr:`~symplyphysics.symbols.basic.mass` of the planet.
+
+Symbol:
+    m
+"""
+
+radial_speed = Symbol("radial_speed", units.velocity)
+"""
+The projection of the velocity vector in the radial direction.
+
+Symbol:
+    v_r
+
+Latex:
+    :math:`v_r`
+"""
+
+potential_energy = Symbol("potential_energy", units.energy)
+"""
+..
+    FIXME Link document below.
```

Comment:
What's this?

Reply:
I can't seem to find a way how to reference another file from within the documentation. I've found there's a `:doc:` directive for that, but it gives an error.

Reply:
I fixed it for you.

Reply:
Thank you, it wasn't obvious.